site/docs/encoding/sort.md

```diff
@@ -55,21 +55,22 @@ If the channel has a discrete scale (`band`, `point` or `ordinal`), the field's
 
 3) Unsorted – `null` – The field is not sorted. This is equivalent to specifying `sort: false` in [Vega's scales](https://vega.github.io/vega/docs/scales/#sort).
 
-4) Specify custom order by:
-  1) providing `sort` with an array that lists valid input domains
-  2) providing custom `scale`'s [`domain`](scale.html#domain)
+4) Specify custom order by providing custom `scale`'s [`domain`](scale.html#domain). (In this case, you don't need to use `sort` property.)
 
-#### Example: Sorting Ordinal Scale by Another Field
+5) Sorting by preferred order of values by providing `sort` with array specifying preferred order of the values. Unspecified values will assume their original orders in the data source, preceded by the orders in the sort array.
```

Comment:
> Sorting by preferred order of values by providing `sort` with array specifying preferred order of the values.

"preferred order of values" showing up twice, quite redundant.

"providing `sort` as an  array specifying preferred order of the values." ?